recipes/outrigger/bld.bat

```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install 
```

Comment:
Please remove `build.sh` and `bld.bat` in favor of using the `script` entry in the meta.yaml file. See:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L26